### PR TITLE
Correction to v1.0 getByIds documentation

### PR DESCRIPTION
**Pull request by Vproject**
The documentation for `getByIds` incorrectly states that full objects are returned. The returned User objects are not complete, though a workaround is possible using `$select` that supplies more complete User objects.
The beta version of this endpoint matches the documentation more closely and supplies more complete User objects without any workarounds.
So perhaps if the relevant parts of the beta version ever make it to v1.0, that would resolve the issue as well. In the meantime I think a documentation update for v1.0 would be helpful to other users of the Graph.

My previous attempt to get this corrected is #2487

### CODE REVIEW

File: api-reference/v1.0/api/directoryobject-getbyids.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ doc_type: apiPageType
 
 # Get directory objects from a list of ids
 
-Returns the directory objects specified in a list of ids.  NOTE: The directory objects returned are the full objects containing **all** their properties. The `$select` query option is not available for this operation.
+Returns the directory objects specified in a list of IDs.
+
+>**Note:** Not all directory objects returned are the full objects containing **all** their properties. For example, [user](../resources/user.md) objects are returned with a limited set of properties. As a temporary workaround, when used in combination with the `$select` query option more complete [user](../resources/user.md) objects will be returned. This behavior is not in accordance with the OData specifications and may be corrected in the future. Therefore this workaround should only be used when you provide `$select=` with all the properties you are interested in and only when future breaking changes to this workaround are acceptable.
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directoryobject-getbyids.md]: I believe a fix is in progress for this.
However if you **really** want to document this behavior and the workaround, I suggest that you put it in the known issues topic and then reference it.

See this example: https://docs.microsoft.com/en-us/graph/api/user-revokesigninsessions?view=graph-rest-1.0&tabs=http and the reference to the known issue (under **Users**): https://docs.microsoft.com/en-us/graph/known-issues?context=graph%2Fapi%2Fbeta&view=graph-rest-beta#users

Comment (Vproject) [reply api-reference/v1.0/api/directoryobject-getbyids.md]: >However if you really want to document this behavior and the workaround, I suggest that you put it in the known issues topic and then reference it.

I've moved it to the known issues, added a reference to it in a Note and mostly restored the original Note.

**Comment by Vproject** [comment]
> Please move your proposed text to the known issues topic.

Has been moved now.

**Comment by Lauragra** [comment]
@Vproject , please sign the CLA so that we can merge this PR. Thanks.

**Comment by Vproject** [comment]
@Lauragra, done.
I had initially missed that a form appears under the CLA text after using the [Sign in with Github to agree] button _and_ giving consent to read part of my Github profile.

**Pull request merged**